gamecreate.php

```diff
@@ -97,6 +97,23 @@
 			throw new Exception(l_t("The phase value is too large or small; it must be between 5 minutes and 10 days."));
 		}
 
+		$input['nextPhaseMinutes'] = (int)$input['nextPhaseMinutes'];
+		if ( $input['nextPhaseMinutes'] < 5 or $input['nextPhaseMinutes'] > 1440*10 or $input['nextPhaseMinutes'] < $input['phaseMinutes'])
+		{
+			throw new Exception(l_t("The next phase value is too large or small; it must be between 5 minutes and 10 days, and must be larger or equal to the value of the original phase value."));
+		}
+
+		if ($input['phaseMinutes'] > 30){

```

Comment:
Could you just add some comments in here explaining why these different logic blocks exist? I'm pretty confident I get them but for any other devs or vDip owners, etc who pull this down having it commented with why is super useful.

Reply:
Added comments.